Sources/SpeziViews/Modules/ConfigureTipKit.swift

```diff
@@ -0,0 +1,53 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Spezi
+@_spi(TestingSupport) import SpeziFoundation
+import TipKit
+
+
+/// Configure TipKit.
+///
+/// This module allows to easily and globally configure TipKit by calling `Tips/configure(_:)`.
+/// You can use the Spezi Dependency system to require TipKit to be configured or can use the `@Environment` property wrapper in your
+/// SwiftUI views to verify that TipKit was configured when using TipKit-based View components.
+public class ConfigureTipKit: Module, DefaultInitializable, EnvironmentAccessible {
```

Comment:
Would be great if we can add a sentence or two about the testing support + preview behavior; maybe as a "note"-box in the docs below the current ones? Would be nice to document and point people to.

Would also be great if we can link to the TipKit Docs.

Reply:
Linked TipKit docs and more details on how testing mode is enabled.

Reply:
Thank you!